apps/backend/src/lib/schema/agri/cancel.ts

```diff
@@ -0,0 +1,98 @@
+import { DOMAIN, VERSION } from "./constants";
+
+export const cancelSchema = {
+  $id: "cancelSchema",
+  type: "object",
+  properties: {
+    context: {
+      type: "object",
+      properties: {
+        domain: {
+          type: "string",
+          enum: DOMAIN
+        },
+        // location: {
```

Comment:
Please check with @tanya-ondc  if this should be commented.

Reply:
Fixed in latest commit